packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -50,477 +50,91 @@ export const pageQuery = graphql`
+If you want to control libraries per file, go to the “Assets” tab on a file. This is
+also where you can drag in components to your file. Click the book icon in the upper
+corner. From the modal, you can turn libraries on and off.
 
-#### Linking the library
+![Buttons to click in Figma to open the library selector](../../../assets/images/design-guidelines/figma-open-library-modal.png)
 
-There are two libraries available in this project:
+![Modal in Figma where you can select the Paste components library to link to your project](../../../assets/images/design-guidelines/figma-link-paste-libraries.png)
 
-- The `console theme` is the current theme you see in Console. Flip the toggle on the Paste docs site to "Current"
-  to see this theme.
-- The `default theme` is the new unified theme, formerly known as the "Unified Design Language" or "UDL". Flip the toggle
-  on the Paste docs site to "Preview" to see this theme.
-
-Link whichever library matches the theme you are using in your project. For example, if you are designing using the
-Default theme, link the `default theme` library to your project.
+The main libraries we recommend you have enabled for each file are:
 
-![Modal in Abstract where you can select libraries from the Paste design tokens project to link to your project](../../../assets/images/design-guidelines/link-theme-library-modal.png)
+- [Paste Components](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components)
+- [Paste Default Theme](https://www.figma.com/file/OZKRQFOkOAl413m5JVIYE6/Paste-Default-Theme) includes tokens (color, text, and shadow styles), needed for the Paste components library
+- [Paste Patterns](https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns)
+- [Brand Illustrations](https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations) includes illustrations for empty states, sections, etc
+- [Asset Documentation](https://www.figma.com/file/GfY11A2VW638eFK48nDtdE/Asset-Documentation) includes helpful utilities for marking up designs, like sticky notes and cursors
+- [File and Cover Sheet Template](https://www.figma.com/file/RfZYRjUH56rZ0qe0veqRJA/File-and-Cover-Sheet-Template)
+- [Console Library](https://www.figma.com/file/vvdKgsO8OLcn7Q6FmuhWtl/Console-Library) for teams with Console projects **(managed by the Console team)**
+- [SendGrid's legacy SG-DS](https://www.figma.com/file/YgSlHVVKgdCY2BzdTbHsPf/SG-DS) for teams with legacy Sendgrid projects **(managed by the Sendgrid team)**
 
-<Callout variant="warning">
-  <CalloutTitle>A note on the SendGrid theme</CalloutTitle>
-  <CalloutText>
-    You will also see the SendGrid theme in this project. If you plan to use this theme, please reach out to us in
-    #help-design-system.
-  </CalloutText>
-</Callout>
+Any time a library is updated, you’ll get a notification in your project asking if you’d like to update.
 
-#### Using the library
+### Using the library
```

Comment:
```suggestion
### Using the Paste Default Theme library
```

Reply:
Above this section, do you mind adding a section on using the Paste Components library? Something short, like so:
### Using the Paste Components library
The Paste Components library houses Figma components for all components available in Paste. You can drop components into your project via the Assets panel on the left, and swap to different variants for a given component via the Design panel on the right. Some more complex components include nested variants, so be sure to explore the different layers on each component for any layer with a ⚙️ emoji. This emoji indicates that there are variants available on that layer. Any layer with a ✏️ emoji indicates an editable layer. Check out our [tips and tricks](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=284%3A1158) for more helpful hints about using the Paste Components library.